docs/src/joint/api/connectionPoints/anchor.html

```diff
@@ -1,4 +1,33 @@
-<p>The <code>'anchor'</code> connection point function is the simplest connection point function. It places the connection point at the link end's anchor point (determined either by the <code>anchor</code> <a href="#anchors">function</a> or by the <code>defaultAnchor</code> <a href="#dia.Paper.prototype.options.defaultAnchor">paper option</a>).
+<p>The <code>'anchor'</code> connection point function is the simplest connection point function. It places the connection point at the link end's anchor point (determined either by the <code>anchor</code> <a href="#anchors">function</a> or by the <code>defaultAnchor</code> <a href="#dia.Paper.prototype.options.defaultAnchor">paper option</a>). It accepts several additional arguments, which can be passed within the <code>connectionPoint.args</code> property:</p>
```

Comment:
```suggestion
<p>The <code>'anchor'</code> places the connection point so that it coincides with the link end's anchor point (determined either by the <code>anchor</code> <a href="#anchors">function</a> or by the <code>defaultAnchor</code> <a href="#dia.Paper.prototype.options.defaultAnchor">paper option</a>). It accepts several additional arguments, which can be passed within the <code>connectionPoint.args</code> property:</p>
```